.github/workflows/dockerd.yml

```diff
@@ -0,0 +1,175 @@
+name: dockerd
+
+on:
+  schedule:
+    - cron: '0 10 * * *'
+  # TODO: add event to build on command in PR (e.g., /test-dockerd)
+  workflow_dispatch:
+    inputs:
+      version:
```

Comment:
version can be a regular release (e.g., `20.10.13`) or a git ref (e.g., `https://github.com/crazy-max/moby.git#buildkit-0.10`)